### PR TITLE
chore: format on mvn install and validate as extra step in gh action

### PR DESCRIPTION
**Pull request by vmutafov**

### CODE REVIEW

**Review by delchev — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -110,10 +110,10 @@
 						</configuration>
 				        <executions>
 							<execution>
-							<id>validate-java-code-formatting</id>
-							<phase>validate</phase>
+							<id>format-java-code</id>
+							<phase>install</phase>
```
Comment (iliyan-velichkov) [inline pom.xml:114]: better format the code before the compilation and tests execution for just in case

**Review by iliyan-velichkov — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (vmutafov) [reply pom.xml]: Fixed. Will do this in the `process-sources` phase

Comment (iliyan-velichkov) [reply pom.xml]: better use `validate`

Comment (vmutafov) [reply pom.xml]: It's not good to use `validate` for such cases. If there is any source generation, `validate` would be too early to format

**Review by ThuF — APPROVED** [review]
+1 Seems good to me.

**Review by delchev — APPROVED** [review]

**Pull request merged**